src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderLineItems.java

```diff
@@ -0,0 +1,21 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderLineItems {
```

Comment:
change class name OrderLineItems to OrderLineItem

Reply:
Changed  to OrderLineItem